apps/heft/src/plugins/TypeScriptPlugin/TypeScriptBuilder.ts

```diff
@@ -915,6 +915,10 @@ export class TypeScriptBuilder extends SubprocessRunnerBase<ITypeScriptBuilderCo
     compilerHost.getDirectories = (folderPath: string) =>
       this._cachedFileSystem.readFolderFilesAndDirectories(folderPath).directories;
 
+    compilerHost.getCurrentDirectory = () => {
+      /* return the build directory from heft config*/
+      return this._configuration.buildFolder;
+    };
```

Comment:
```suggestion
    /* Use the Heft config's build folder because it has corrected casing */
    compilerHost.getCurrentDirectory = () => this._configuration.buildFolder;
```